label_sync/kubeflow_label.yml

```diff
@@ -102,8 +102,8 @@ labels:
   name: status/icebox
 - color: ffa500
```

Comment:
Should we change the status/approved label to approved and also add label lgtm? Those labels are used by the k8s-ci-bot. So status/approved is duplicative. I think by adding them here we can ensure consistent colors.